### DISCUSSION TITLE
How can I list all my variant-type products in the grid?

### DISCUSSION THREAD

**Discussion by AlephLau** [pimcore/pimcore#16854] [Q&A / Support] [answered]
In Pimcore 10, I did this using the SQL filter functionality for the data object grid, so that it would list all the variants from the "type" column. However, this functionality has been removed in Pimcore 11, and when I look at the options available in the grid options under system columns, the "type" column is not listed. Is there any way to list only my variants in the grid for all my product-type objects?

![image](https://github.com/pimcore/pimcore/assets/70108013/209acaac-acfd-4335-8244-4490f8a56474)

**Comment by AlephLau** [comment]
@mattamon Can you help me with that pls?

**Reply by mattamon** [reply, marked as answer]
Hey @AlephCode 

I tried to achieve the describe result, but no luck so far. You can get the value pretty easily with formatters and extractors e.g. "Operator Any Getter" but there is no way to filter those columns then. 

Could you maybe open up an issue for this, so we can investigate that we either show the type in the system columns or that you can filter generated columns? Grid stuff should be considered in the https://github.com/pimcore/admin-ui-classic-bundle

**Reply by AlephLau** [reply]
@mattamon  This week I will open an issue explaining this on https://github.com/pimcore/admin-ui-classic-bundle. Thank you very much for your help.
